### PR TITLE
create checks for (grub2|uefi)_no_removeable_media

### PR DESCRIPTION
**Pull request by vojtapolasek**
#### Description:

Created oval checks for rules grub2_no_removeable_media and uefi_no_removeable_media. Created test only for one instance as the checks are the same, only the file path is different. Fixed rule titles, although Boat Loader is nice, it is not correct.

#### Rationale:

Rules were missing checks.

### CODE REVIEW

File: linux_os/guide/system/bootloader-grub2/grub2_no_removeable_media/rule.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ documentation_complete: true
 
 prodtype: wrlinux1019,rhel7,rhel8,ol7,ol8,rhv4
 
-title: 'Boat Loader Is Not Installed On Removeable Media'
+title: 'Boot Loader Is Not Installed On Removeable Media'
```
Comment (jan-cerny) [inline linux_os/guide/system/bootloader-grub2/grub2_no_removeable_media/rule.yml]: should be removable

Comment (vojtapolasek) [reply linux_os/guide/system/bootloader-grub2/grub2_no_removeable_media/rule.yml]: ACtually maybe not necessarily:
https://www.collinsdictionary.com/dictionary/english/removeable

**Comment by jan-cerny** [comment]
@vojtapolasek Is a remediation planned to be created? If there will be no remediation, please add `# remediation = none` to the test scenarios at the beginning.

Otherwise it's great.

**Comment by vojtapolasek** [comment]
I don't think that remediation is planed, we don't know the user's machine setup and there can be many variants.

**Pull request merged**